src/sage/groups/abelian_gps/abelian_group.py

```diff
@@ -458,10 +458,16 @@ def is_AbelianGroup(x):
         sage: F = AbelianGroup(5,[5,5,7,8,9], names=list("abcde")); F
         Multiplicative Abelian group isomorphic to C5 x C5 x C7 x C8 x C9
         sage: is_AbelianGroup(F)
+        doctest:warning...
+        DeprecationWarning: the function is_AbelianGroup is deprecated;
+        use 'isinstance(..., AbelianGroup_class)' instead
+        See https://github.com/sagemath/sage/issues/37898 for details.
         True
         sage: is_AbelianGroup(AbelianGroup(7, [3]*7))
         True
     """
+    from sage.misc.superseded import deprecation
+    deprecation(37898, "the function is_AbelianGroup is deprecated; use 'isinstance(..., AbelianGroup_class)' instead")
```

Comment:
shouldn't that be `use 'in Groups.Commutative'`, @fchapoton?

Reply:
I just checked, yes, that's the pattern.  For example:
```
deprecation(35999, "the function is_CommutativeAlgebra is deprecated; use '... in Algebras(base_ring).Commutative()' instead")
```
or
```
deprecation(35253, "the function is_Algebra is deprecated; use '... in Algebras(base_ring)' instead")
```

Reply:
I did use this pattern in these other deprecation messages when there was at least some indication that these `is_...` functions were used with this meaning.
But `is_AbelianGroup` has never been used with a meaning other than this specific `isinstance` test.